api/ClipboardItem.json

```diff
@@ -203,6 +203,7 @@
       "presentationStyle": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/ClipboardItem/presentationStyle",
+          "spec_url": "https://w3c.github.io/clipboard-apis/#dom-clipboarditem-presentationstyle",
```

Comment:
What's with the rest of this API?

Reply:
> What's with the rest of this API?

`presentationStyle` is the only member that has an actual MDN article. The object itself and all other other members aren’t documented in MDN yet.

Reply:
hm, but there are pages:
https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem
https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/ClipboardItem
https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/getType
https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/types

Reply:
> hm, but there are pages:
> [developer.mozilla.org/en-US/docs/Web/API/ClipboardItem](https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem)
> [developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/ClipboardItem](https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/ClipboardItem)
> [developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/getType](https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/getType)
> [developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/types](https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/types)

Indeed yeah — thanks for catching those. The cause was that the script I run for scraping the spec URLs wasn’t catching pages that had been added relatively recently (the last ~3 months or so). But  I’ve fixed it now and pushed an additional commit here — which, along with adding the ClipboardItem spec URLs, also adds them for CharacterData, ContactsManager, and CookieChangeEvent.

(And I’ll also go back now and see what other pages got added recently that the script missed — and then raise other PRs as needed for those.)

Reply:
Great! Glad we spotted this before going further into the world of API docs. Happy to review PRs for those missed spec_urls